src/app/features/exchange-set/ess-upload-file/ess-upload-file/ess-upload-file.component.scss

```diff
@@ -0,0 +1,7 @@
+.UploadFileIput{
```

Comment:
I notice .UploadFileInput and .UploadButton styles are not nested inside .uploadENCFileSection. Please nest the styles if it's better practice to do so.

Reply:
Resolved